geopandas/io/file.py

```diff
@@ -123,17 +123,16 @@ def to_file(df, filename, driver="ESRI Shapefile", schema=None, **kwargs):
 
 
 def infer_schema(df):
-    try:
-        from collections import OrderedDict
-    except ImportError:
-        from ordereddict import OrderedDict
+    from collections import OrderedDict
 
     def convert_type(column, in_type):
         if in_type == object:
             return "str"
         if in_type.name.startswith("datetime64"):
             # numpy datetime type regardless of frequency
             return "datetime"
+        if in_type == "Int64":
+            return "int"
```

Comment:
We can probably also add a "string" -> "str" conversion for the new string dtype (only in master), and actually also "boolean" -> "bool" (that's only in a PR). 

So maybe we should keep a dictionary of conversions for those?

Reply:
Make sense. There might be more in the future, dict will make it expandable. Will do.